system/jlib/jsocket.cpp

```diff
@@ -1969,11 +1982,13 @@ void CSocket::readtms(void* buf, size32_t min_size, size32_t max_size, size32_t
                 }
                 else
                 {
-                    if (nonblocking && (err == JSE_WOULDBLOCK || err == EAGAIN)) // if EGAIN or EWOULDBLOCK - no more data to read
+                    if (err == JSE_WOULDBLOCK || err == EAGAIN) // if EGAIN or EWOULDBLOCK - no more data to read
```

Comment:
spelling in comment - EGAIN
Should we add a JSE_EAGAIN ?

Reply:
yeah, I'll add JSE_EGAIN for completeness.